cdap-ui/app/directives/dag-plus/my-dag-ctrl.js

```diff
@@ -39,10 +39,11 @@ angular.module(PKG.name + '.commons')
 
     var metricsPopovers = {};
     var selectedConnections = [];
-    var endpointClicked = false;
-    var connectionDropped = false;
     var dagMenu;
     let conditionNodes = [];
+    var alertNodes = [];
+    var errorNodes = [];
+    let normalNodes = [];
```

Comment:
Where do we use these arrays? We seem to be pushing nodes into this and resetting them to empty array in `resetEndpointsAndConnections` functions. Do we need these?